src/modules/booking/components/booking-form/booking-form.tsx

```diff
@@ -0,0 +1,92 @@
+import { type FC } from 'react';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+// components
+import { TextField } from '@/modules/core/components/text-field';
+import { Button } from '@/modules/core/components/button';
+// type
+import type {
+  BookingFormValue,
+  BookingFormProps,
+} from './booking-form.interface';
+// styles
+import styles from './booking-form.module.scss';
+
+const defaultValues = {
+  name: '',
+  lastName: '',
+  phone: '',
+  comment: '',
+};
+
+const validationSchema: z.Schema<BookingFormValue> = z.object({
+  name: z.string().min(1),
+  lastName: z.string().min(1),
+  phone: z.string().regex(/^\+\d{1,3}\d{10}$/, 'Phone number is not valid'),
+  email: z.string(),
+  comment: z.string(),
+});
+
```

Comment:
Instead of writing types you can just infer it from the zod itself

```
const validationSchema = z.object({
  name: z.string().min(1),
  lastName: z.string().min(1),
  phone: z.string().regex(/^\+\d{1,3}\d{10}$/, 'Phone number is not valid'),
  email: z.string(),
  comment: z.string(),
});

export type BookingFormValue= z.infer<typeof validationSchema>;
```